src/hyper_tokio/connector.rs

```diff
@@ -21,35 +24,36 @@ use url::Url;
 
 #[derive(Clone, Debug)]
 pub struct SlackClientHyperConnector<H: Send + Sync + Clone + connect::Connect> {
-    hyper_connector: Client<H>,
+    hyper_connector: Client<H, Body>,
     tokio_rate_controller: Option<Arc<SlackTokioRateController>>,
 }
 
-pub type SlackClientHyperHttpsConnector = SlackClientHyperConnector<HttpsConnector<HttpConnector>>;
+pub type SlackClientHyperHttpsConnector =
+    SlackClientHyperConnector<HttpsConnector<connect::HttpConnector>>;
 
-impl SlackClientHyperConnector<HttpsConnector<HttpConnector>> {
-    pub fn new() -> Self {
+impl SlackClientHyperConnector<HttpsConnector<connect::HttpConnector>> {
+    pub fn new() -> std::io::Result<Self> {
```

Comment:
This will be a small but breaking change. Should we use `.unwrap()` here and migrate to a new method something like `::try_new()`?

Reply:
I think we will need to release v2, since a lot of things changed including external deps. So, probably it is fine to leave it.